extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+/* Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
We created that file afresh for a4a-core, but copy-pasted the notice.  We noticed and updated the date now.  We can put it back to 2015 if you think that's more appropriate.